src/Testable/Internal.elm

```diff
@@ -2,12 +2,14 @@ module Testable.Internal exposing (..)
 
 import Http
 import Time exposing (Time)
+import Platform.Cmd
 
 
 type Cmd msg
     = None
     | TaskCmd (Task msg msg)
     | Batch (List (Cmd msg))
+    | PortCmd (Platform.Cmd.Cmd msg)
```

Comment:
I totally agree, I will then:
- Change `PortCmd` to `WrappedCmd`
- Remove `Testable/Port.elm`, moving `wrap` function from `Testable/Port.elm` to `Testable/Cmd.elm`
- Rename `containsPortCmd` to `containsCmd`
- Rename `assertPortCalled` to `assertCmdCalled`

Do you agree? Feel free to suggest better namings